components/camel-spring-ws/src/main/java/org/apache/camel/component/spring/ws/SpringWebserviceProducer.java

```diff
@@ -139,7 +149,18 @@ private void populateMessageHeaderFromResponse(Message message, SoapHeader soapH
 
         }
     }
-
+    /**
+     * Populates message attachments from soap response attachments 
+     * @param inOrOut {@link Message}
+     * @param soapMessage {@link SoapMessage}
+     */
+    private void populateMessageAttachmentsFromResponse(Message inOrOut, Iterator<Attachment> attachments) {
```

Comment:
Looks like you may have checkstyle issues. Can you run 

   mvn compile -Psourcecheck